.github/workflows/mvn_docker_image.yml

```diff
@@ -88,20 +88,35 @@ on:
             jib_custom_docker_image_repo_password:
                 required: false
                 description: custom docker image repo password for for docker build.
+            self_hosted_cache_endpoint:
```

Comment:
Why not as input? It's not secret anyway. Same for `self_hosted_cache_port`, `self_hosted_cache_bucket` and `self_hosted_cache_region`. Then it would also be possible to already set default values which makes using this action easier

Reply:
This change belongs to PR https://github.com/UbiqueInnovation/workflows-backend/pull/69 I changed the merge base and changed it there